### DISCUSSION TITLE
Dashboard Datasource - ability to hide the panel, or query a panel from another dashboard

### DISCUSSION THREAD

**Discussion by aengusrooneygrafana** [grafana/grafana#38329] [Data Sources]
**What would you like to be added**: 

`Dashboard Datasource`   https://grafana.com/docs/grafana/latest/datasources/#supported-data-sources 

For the Dashboard Datasource, we would like to have the ability to hide the panel, or query the panel from another dashboard.  

**Why is this needed**:

A similar functionality is available in Splunk, we would like to replicate this 'feature' in our Grafana dashboards.  In Splunk, the 'base search' can be hidden, so users can not access the 'raw' data.  

https://docs.splunk.com/Splexicon:Basesearch 

https://docs.splunk.com/Documentation/Splunk/8.1.0/Viz/Savedsearches 
    → See "Dashboard Source Code Example (Basesearch)"

Please let me know if further information is required.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
